pkgs/development/tools/mcap/mcap-cli/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, buildGoModule
+, fetchgit
+}:
+
+buildGoModule rec {
+  pname = "mcap-cli";
+  version = "0.0.31";
+
+  src = fetchgit {
+    url = "https://github.com/foxglove/mcap";
+    rev = "releases/mcap-cli/v${version}";
+    sha256 = "sha256-SmI2MSKf4j6u+MnO1WH4s8i9H9kpUkjIhhrwk/qht2s=";
+    fetchLFS = true;
+  };
+
+  sourceRoot = "mcap/go/cli/mcap";
+
+  vendorHash = "sha256-3TSwOgTIaBg5U8UdQ74LmsS8gS05Yw02JjfaIN9fKQQ=";
+
+  GOWORK = "off";
+
+  # Tests in upstream mcap cli are currently failing:
+  # https://github.com/foxglove/mcap/issues/895
```

Comment:
If you've got the tests to work, this comment can probably be removed?

Reply:
fixed